webrender/build_gfx.rs

```diff
@@ -341,20 +331,9 @@ fn extend_sampler_definition(
 }
 
 fn replace_non_sampler_uniforms(new_data: &mut String) {
-    new_data.push_str(
-        "\tlayout(push_constant) uniform PushConsts {\n\
-         \t\tmat4 uTransform;       // Orthographic projection\n\
-         \t\t// A generic uniform that shaders can optionally use to configure\n\
-         \t\t// an operation mode for this batch.\n\
-         \t\tint uMode;\n\
-         \t} pushConstants;\n",
-    );
     new_data.push_str(&format!(
-        "\tlayout(set = {}, binding = 0) uniform Locals {{\n\
+        "\tlayout(set = {}, binding = 0) uniform Projection {{\n\
          \t\tuniform mat4 uTransform;       // Orthographic projection\n\
-         \t\t// A generic uniform that shaders can optionally use to configure\n\
-         \t\t// an operation mode for this batch.\n\
-         \t\tuniform int uMode;\n\
          \t}};\n",
          DESCRIPTOR_SET_LOCALS
```

Comment:
what is this parameter for?

Reply:
This is for the format string: https://github.com/szeged/webrender/pull/346/files/58e78c1b0097e66e1d0656a0ff86e8eb0ddc3c25#diff-711aa321dbe62813ad158403a418e1d3R335